index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Adrian and Faruk's Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+ 
+
+ 
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
The script is supposed to be in the header, not in the body. Make sure you pay attention next time 👍